src/main/java/com/autotune/analyzer/recommendations/RecommendationConstants.java

```diff
@@ -738,6 +738,9 @@ public static class PercentileConstants {
             public static final Integer TWENTYFIVE_PERCENTILE = 25;
             public static final Integer SEVENTYFIVE_PERCENTILE = 75;
             public static final Integer FIFTY_PERCENTILE = 50;
+            public static final Integer COST_ACCELERATOR_PERCENTILE = 60;
+            public static final Integer PERFORMANCE_ACCELERATOR_PERCENTILE = 100;
```

Comment:
We are using 98th percentile for CPU/Mem, this has to be the same

Reply:
Done @dinogun